Detector/DetFCChhBaseline1/compact/FCChh_DectDimensions.xml

```diff
@@ -65,58 +65,55 @@
     <constant name="FwdTracker_z2" value="2000*mm"/>
     <constant name="FwdTracker_zOffset" value="14000*mm"/>
 
-    <!-- E(M)CAL DIMENSIONS -->
-    <!-- E(M)Cal Barrel -->
+    <!-- CALORIMETER DIMENSIONS -->
+    <!-- ECal Barrel (EM B) -->
     <constant name="BarECal_id" value="5"/>
     <constant name="BarECal_rmin" value="1750*mm"/>
     <constant name="BarECal_rmax" value="2750*mm"/>
     <constant name="BarECal_dz" value="5000*mm"/>
-    <!-- E(M)Cal End-caps -->
-    <constant name="EndCapECal_id" value="6"/>
-    <constant name="EndCapECal_rmin1" value="864.278*mm"/>
-    <constant name="EndCapECal_rmax1" value="2800*mm"/>
-    <constant name="EndCapECal_rmin2" value="1467.641*mm"/>
-    <constant name="EndCapECal_rmax2" value="2800*mm"/>
-    <constant name="EndCapECal_dz" value="1850*mm"/>
-    <constant name="EndCapECal_zOffset" value="5300*mm + EndCapECal_dz"/>
-    <!-- E(M)Cal forward -->
-    <constant name="FwdECal_id" value="7"/>
-    <constant name="FwdECal_rmin" value="30*mm"/>
-    <constant name="FwdECal_rmax" value="3500*mm"/>
-    <constant name="FwdECal_dz" value="750*mm"/>
-    <constant name="FwdECal_zOffset" value="16500*mm+FwdECal_dz"/>
-
-
-    <!-- HCAL DIMENSIONS -->
-    <!-- HCal Barrel -->
+    <!-- HCal Barrel (HCAL B) -->
     <constant name="BarHCal_id" value="8"/>
     <constant name="BarHCal_rmin" value="2850*mm"/>
-    <constant name="BarHCal_rmax" value="4850*mm"/>
+    <constant name="BarHCal_rmax" value="4890*mm"/>
```

Comment:
Hi Joschka, that's exactly it. It is a new requirement from HCAL, as the support is a bit bigger than we thought it would be.